libs/libx264/Makefile

```diff
@@ -8,55 +8,49 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=x264
-PKG_VERSION:=snapshot-20181006-2245
+PKG_VERSION:=snapshot-20190324-2245
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://download.videolan.org/x264/snapshots/
 PKG_MAINTAINER:=Adrian Panella <ianchi74@outlook.com>
-PKG_HASH:=eae60e969958c0759554d5e338dea9b91f1f1fe85d9523c152c2259d8546f469
+PKG_HASH:=68010057edaadffc7593933d13084e8d32e041c42b17c089513d88c917f2ad54
 
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
 
-PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
-TARGET_CFLAGS:=$(filter-out -O%,$(TARGET_CFLAGS)) -std=gnu99 -fPIC -O3 -ffast-math -I.
+TARGET_CFLAGS := $(filter-out -O%,$(TARGET_CFLAGS))
 MAKE_FLAGS+= LD="$(TARGET_CC) -o"
 
-# ARM ASM depends on ARM1156 or later, blacklist earlier or incompatible cores
-# AMD Geode LX and i486 do not have SSE
-CPU_ASM_BLACKLIST:=geode i486 arm920t arm926ej-s arm1136j-s arm1176jzf-s fa526 mpcore xscale \
-		   464fp mips32 24kc 34kc 74kc octeon mips64
+# Instead of blacklisting a boatload of platforms just enable
+# optimization where it makes sense (matters) ie ARMv7+ and x86_64
 
-ifneq ($(CONFIG_SOFT_FLOAT)$(findstring $(call qstrip,$(CONFIG_CPU_TYPE)),$(CPU_ASM_BLACKLIST)),)
-  CONFIGURE_VARS+= AS= 
-  MAKE_FLAGS+= AS= 
-  CONFIGURE_ARGS += --disable-asm
-else
+ifneq ($(findstring cortex-a,$(CPU_TYPE)),)
   CONFIGURE_ARGS += --enable-lto
-ifneq ($(CONFIG_TARGET_x86),)
-ifeq ($(CONFIG_NASM),y)
-  CONFIGURE_VARS+= AS=nasm
-  MAKE_FLAGS+= AS=nasm
-else
-  CONFIGURE_VARS+= AS= 
-  MAKE_FLAGS+= AS= 
-  CONFIGURE_ARGS += --disable-asm
-endif
-endif
+  else
+  ifneq ($(CONFIG_TARGET_x86_64),)
+    CONFIGURE_ARGS += --enable-lto
+    ifeq ($(CONFIG_NASM),y)
+      CONFIGURE_VARS+= AS=nasm
+      MAKE_FLAGS+= AS=nasm
+    endif
+    else
+    CONFIGURE_ARGS += --disable-asm
+  endif
 endif
 
 CONFIGURE_ARGS += \
+		--disable-cli \
```

Comment:
I'd like this sorted by --enable first and then --disable. Alphabetical as well.

Reply:
They're sorted by the output of ```./configure --help``` to easier keep track of changes from upstream.